app/controllers/organizations/activations_controller.rb

```diff
@@ -1,23 +1,37 @@
 module Organizations
   class ActivationsController < Organizations::BaseController
-    def update
-      @user = User.find(params[:user_id])
-
-      authorize! @user, with: ActivationsPolicy
+    before_action :set_pg
 
-      if @user.deactivated_at
-        @user.activate
+    def update
+      @result = if params[:person_group][:deactivated] == "true"
```

Comment:
I don't think `@result` needs to be an instance variable here? I don't see it used in the view.